<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>621</width>
      <height>39</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLineEdit" name="searchBar">
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
        </font>
       </property>
       <property name="placeholderText">
        <string>Enter city name...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
        </font>
       </property>
       <property name="text">
        <string>Search City</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cityCombo">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="currentCity">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 oblique 14pt &quot;DejaVu Sans Mono&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>150</y>
      <width>951</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="picture">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>101</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="temperature">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>101</width>
          <height>71</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
          <pointsize>17</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="day">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>10</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 oblique 11pt &quot;DejaVu Sans Mono&quot;;</string>
        </property>
        <property name="text">
         <string>Today</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="picture_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>101</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="temperature_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>101</width>
          <height>71</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
          <pointsize>17</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="day_2">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>10</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 oblique 11pt &quot;DejaVu Sans Mono&quot;;</string>
        </property>
        <property name="text">
         <string>Today</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="picture_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>101</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="temperature_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>101</width>
          <height>71</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
          <pointsize>17</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="day_3">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>10</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 oblique 11pt &quot;DejaVu Sans Mono&quot;;</string>
        </property>
        <property name="text">
         <string>Today</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QFrame" name="frame_4">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="picture_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>101</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="temperature_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>101</width>
          <height>71</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
          <pointsize>17</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="day_4">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>10</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 oblique 11pt &quot;DejaVu Sans Mono&quot;;</string>
        </property>
        <property name="text">
         <string>Today</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QFrame" name="frame_5">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="picture_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>101</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="temperature_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>101</width>
          <height>71</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
          <pointsize>17</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="day_5">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>10</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 oblique 11pt &quot;DejaVu Sans Mono&quot;;</string>
        </property>
        <property name="text">
         <string>Today</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QFrame" name="frame_6">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="picture_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>101</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="temperature_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>101</width>
          <height>71</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
          <pointsize>17</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="day_6">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>10</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 oblique 11pt &quot;DejaVu Sans Mono&quot;;</string>
        </property>
        <property name="text">
         <string>Today</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QFrame" name="frame_7">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="picture_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>101</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="temperature_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>101</width>
          <height>71</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
          <pointsize>17</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="day_7">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>10</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 oblique 11pt &quot;DejaVu Sans Mono&quot;;</string>
        </property>
        <property name="text">
         <string>Today</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
